Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCMediaEndpoint.cpp

```diff
@@ -82,11 +82,13 @@ LibWebRTCMediaEndpoint::LibWebRTCMediaEndpoint(LibWebRTCPeerConnectionBackend& p
     ASSERT(isMainThread());
     ASSERT(client.factory());
 
-    String fieldTrials = "WebRTC-Video-H26xPacketBuffer/Enabled/"_s;
+#define WEBKIT_WEBRTC_FIELD_TRIALS "WebRTC-Video-H26xPacketBuffer/Enabled/"
+    const char* fieldTrials = WEBKIT_WEBRTC_FIELD_TRIALS;
```

Comment:
We should use ASCIILiteral in new code.

Reply:
I'm not sure that I understand.  `webrtc::field_trial::InitFieldTrialsFromString()` below takes a `const char*`, so how is using an `WTF::ASCIILiteral` going to help us here?  We're going to create the C++ wrapper object, then pull out the `const char*` again to pass it to the function...?

Reply:
Just trying to follow our new safer C++ guidelines:
- https://github.com/WebKit/WebKit/wiki/Safer-CPP-Guidelines#use-asciiliteral-instead-of-const-char-for-string-literals

The benefits in this particular instance are likely very small but it is good practice in general.

Reply:
Okay, I have a fix.